src/components/LogRow/BaseRow/index.tsx

```diff
@@ -128,13 +133,26 @@ const ProcessedBaseRow: React.FC<ProcessedBaseRowProps> = memo((props) => {
         (match) => `<mark>${escapeHtml(match)}</mark>`
       );
     }
+    if (highlights) {
+      const shouldCheckForOverlappingRegex = searchTerm !== undefined;
+      const hasOverlappingRegexes =
+        shouldCheckForOverlappingRegex &&
+        hasOverlappingRegex(searchTerm, highlights, children);
+
+      if (!hasOverlappingRegexes) {
+        render = render.replace(
+          highlights,
+          (match) => `<mark>${escapeHtml(match)}</mark>`
+        );
```

Comment:
I think there might be a small bug with the `replace` call, because it only highlights the first instance of the word rather than all of them. In the screenshot below, I've added `uuid` as a highlight but only the first instance in each line gets highlighted. 

<img width="1023" alt="Screen Shot 2022-10-28 at 11 29 35 AM" src="https://user-images.githubusercontent.com/47064971/198675558-744f29d4-9b3e-472b-9838-413da792cc6c.png">


This differs from Lobster's behavior, which highlights all instances of the highlight word in each line (the highlight word is `project` here):

<img width="921" alt="Screen Shot 2022-10-28 at 11 28 54 AM" src="https://user-images.githubusercontent.com/47064971/198675442-70db7e2e-68ff-4ee5-9415-4ad1ca811222.png">

I think this also holds true for search terms.

Reply:
Good catch

Reply:
Great catch i fixed this!